From 220c5bf347ee395f7b0ef3eb237e88d35fb7e212 Mon Sep 17 00:00:00 2001
From: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Date: Thu, 13 Feb 2025 03:11:46 +0200
Subject: [PATCH] Makefile: provide 'clean' target

It's useful to provide a 'make clean' target as a lot of scripts expect
to be able to run it. Add empty 'clean' target to make those scripts
happy.

Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Upstream-Status: Backport [https://github.com/linux-msm/dsp-binaries/commit/220c5bf347ee395f7b0ef3eb237e88d35fb7e212]
---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 7cc2f67a5b98..67006ff79ee9 100644
--- a/Makefile
+++ b/Makefile
@@ -5,6 +5,8 @@ DSPDIR = ${prefix}/share/qcom
 
 all:
 
+clean:
+
 install:
 	./scripts/install.sh config.txt ${DESTDIR}/${DSPDIR}
 
-- 
2.47.2

